cmds/truncate/truncate.go

```diff
@@ -36,6 +37,13 @@ func init() {
 		os.Args[0] = cmd
 		defUsage()
 	}
+
+	u, err := unit.NewUnit(unit.DefaultUnits)
+	if err != nil {
+		log.Fatal(err)
+	}
+	size = *u.MustNewValue(1, unit.None)
```

Comment:
nit: no need for *

this could also just be

size = unit.MustNewUnit(unit.DefaultUnits).MustNewValue(1, unit.None)